src/types/UiaTracing.cpp

```diff
@@ -126,6 +126,140 @@ inline std::wstring UiaTracing::_getValue(const TextUnit unit) noexcept
     }
 }
 
+std::wstring UiaTracing::convertAttributeId(const TEXTATTRIBUTEID attrId) noexcept

```

Comment:
Do these need to construct strings? Especially since they're going into ETW channel stuff... it's probably a perf issue to construct them. Reason being that this will be evaluated, it'll construct the string and return, and then the ETW macro will realize no one is listening and then throw it all away so the work was for nothing.

Can we return `wstring_view&` references to some sv constants so they're not constructed again and again? 

cc: @lhecker

Reply:
It seems like this function in particular is only called when ETW is enabled, right?
In that case I wouldn't bother making so many statics and just return a `std::wstring_view`.

Reply:
I liked Dustin's idea of just removing it entirely haha. Much easier :P